config/ftbquests/quests/chapters/futuristic_mechanisms.snbt

```diff
@@ -1714,6 +1713,20 @@
 			x: -4.0d
 			y: -6.0d
 		}
+		{
+			dependencies: ["6131A76D203DE6AB"]
+			description: ["These can replace your old Cleanroom Filters, and turn it into a Sterile Cleanroom."]
```

Comment:
Someone found out that the Sterile Cleanroom can't serve as a normal Cleanroom. It was confirmed as intended by a GTCEu dev in this closed issue: https://github.com/GregTechCEu/GregTech-Modern/issues/1154